docs/types/fields.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Fields
+---
+
+# Fields
+
+## Usage
+
+GraphQL fields can be declared in Strawberry using either
+[dataclass][dataclass_docs]-style fields, or using the `strawberry.field`
+function. The function can be used as a decorator around a resolver for the
+field, or directly provided to the function directly using its `resolver`
+argument.
```

Comment:
we can also use without resolver, for example to change the name of the field. Should we mention that?

Reply:
I think the whole sentence needs rewording actually...